frontend/src/context/user-context.js

```diff
@@ -29,26 +28,26 @@ export function UserContextProvider(props) {
     await axios
       // Must set "withCredentials" to true so the cookie stored on browser
       // is sent to the server for checking
-      .get(URL_USER_SVC_JWT_VERIFICATION, { withCredentials: true })
+      .get(URL_AUTH_SVC_JWT_VERIFICATION, { withCredentials: true })
       .then((res) => {
         const isAuthSuccess = res && res.status === STATUS_CODE_OK;
         setIsSignedIn(isAuthSuccess);
         callback(isAuthSuccess, null);
         return res.data;
       })
-      .then((data) => setUsername(data.username))
+      .then((data) => setUsername(data.user))
```

Comment:
Can I pardon you to use .username instead to avoid confusion? Thanks!

Reply:
Using user instead of username here because the JWT actually contains username OR email depending on which was used to sign in.

Reply:
I see, then can you help making it to get the username instead? Since it will be used in communication service to display the sender username

Reply:
Ah I see, sure.